### ISSUE TITLE
rc2nix string literal is not escaped

### ISSUE THREAD

**Issue by luochen1990** [nix-community/plasma-manager#13]
I find following contents in the output of rc2nix:

```
"khotkeysrc"."Data_3Conditions0Window0"."Class" = "^konqueror\s";
```

and try `"^konqueror\s"` in nix repl will give you `"^konquerors"`, which might cause error (I'm not sure)

**Comment by pjones** [comment]
You are correct, this looks like a bug in `rc2nix`.  It should properly escape the string but doesn't.
